packages/ember-routing/lib/system/router.js

```diff
@@ -1011,7 +1011,7 @@ function appendLiveRoute(liveRoutes, route, parentRoute, defaultParentState, ren
   var target;
   var myState = {
     render: renderOptions,
-    outlets: Object.create(null)
+    outlets: Ember.create(null)
```

Comment:
This is available as a utility:

```javascript
import o_create from "ember-metal/platform/create";
```

Example: https://github.com/emberjs/ember.js/blob/995ed0ae7ac9d7edd8305f1a634404a2d0c5e0d9/packages/ember-htmlbars/lib/helpers.js#L6